.rubocop_todo.yml

```diff
@@ -0,0 +1,21 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2018-10-26 17:43:57 +0200 using RuboCop version 0.60.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: EnforcedStyleForLeadingUnderscores.
+# SupportedStylesForLeadingUnderscores: disallowed, required, optional
+Naming/MemoizedInstanceVariableName:
+  Exclude:
+    - 'lib/jekyll-redirect-from/page_without_a_file.rb'
```

Comment:
There's point marking this as a `TODO`.. It can never be "fixed"..
Best we add an inline comment to disable-reenable the cop

Reply:
I don't like inline comments, I'd rather keep it here, even if it's not "fixable".